packages/clay-card/src/CardWithInfo.tsx

```diff
@@ -83,7 +89,12 @@ export interface IProps extends React.BaseHTMLAttributes<HTMLDivElement> {
 	/**
 	 * Callback for when item is selected
 	 */
-	onSelectChange?: (val: boolean) => void;
+	onSelectChange?: (val: string | boolean) => void;
```

Comment:
My only concern is that this may break the current components due to the new type... I'm thinking about how we can avoid this.

Reply:
Hey @matuzalemsteles! in order not to break the current components, I have added that the `onSelectChange` type will be different depending on whether the `selectableType` is radio or checkbox (de0d533de063f132d9ef65b21e9584abba0a3860). Let me know me what you think and if you think it's okay for you, I'll do the same for the rest of the cards.

Reply:
In another vein, the tests are failing because the clay-card threshold is not being met (in this PR the value is 88.88% not 90%). Is there any parameter that you take into account to increase this?

Reply:
> Let me know me what you think and if you think it's okay for you

Oh great, that will solve it. Thanks!

> In another vein, the tests are failing because the clay-card threshold is not being met (in this PR the value is 88.88% not 90%). Is there any parameter that you take into account to increase this?

We don't actually have a set value, but we always try to keep it above 50%, and we always increase it when coverage also increases, it's just a way to always keep coverage high. In this case, as you added new test cases to cover the behavior, you can lower the threshold without any problems.